docs/7___appendix.adoc

```diff
@@ -51,10 +51,11 @@ For details see <<MS93>>.
 
 |_event_
 |Something that occurs instantaneously at a specific time or when a specific condition occurs.
-At an event, numerical integration is suspended and variables may change their values discontinuously.
-Internal events occur inside the FMU and should be signaled to the environment without any delay and can cause event handling and/or the activation of an output clock.
-Input clocks are activated by the environment to inform the FMU about the exact moment an event across FMUs has to be handled.
-See <<state-event>>, <<step-event>> and <<time-event>>.
+At an event in Model Exchange, numerical integration is suspended and variables may change their values discontinuously.
+In Co-Simulation, an event may interrupt a <<fmi3DoStep>> to introduce an additional communication point.
+Internal events occur inside the FMU and should be signaled to the environment without any delay (in CS using <<early-return,early return>>, in SE using <<IntermediateUpdateMode,intermediate update>>) and can cause event handling and/or the activation of an output clock.
+Input clocks are activated by the environment to inform the FMU about the exact moment an event has to be handled by the FMU.
```

Comment:
I used "point in time" in similar situations instead of "the exact moment".

Reply:
accepted